pom.xml

```diff
@@ -205,9 +205,9 @@
         <version>${com.mattbertolini.version}</version>
       </dependency>
       <dependency>
-        <groupId>com.smartgwt</groupId>
-        <artifactId>smartgwt</artifactId>
-        <version>${com.smartgwt.version}</version>
+        <groupId>com.isomorphic.smartgwt.lgpl</groupId>
```

Comment:
Please re-order the maven dependency declaration to keep the alphabetical order by groupId